projects/elfutils/build.sh

```diff
@@ -93,11 +93,20 @@ $CXX $CXXFLAGS $LIB_FUZZING_ENGINE fuzz-dwfl-core.o \
 $CC $CFLAGS \
   -D_GNU_SOURCE -DHAVE_CONFIG_H \
   -I. -I./lib -I./libelf -I./libebl -I./libdw -I./libdwelf -I./libdwfl -I./libasm \
-  -c "$SRC/fuzz-elf-get-sections.c" -o fuzz-elf-get-sections.o
-$CXX $CXXFLAGS $LIB_FUZZING_ENGINE fuzz-elf-get-sections.o \
+  -c "$SRC/fuzz-libelf.c" -o fuzz-libelf.o
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE fuzz-libelf.o \
 	./libasm/libasm.a ./libebl/libebl.a ./backends/libebl_backends.a ./libcpu/libcpu.a \
   ./libdw/libdw.a ./libelf/libelf.a ./lib/libeu.a -l:libz.a \
-	-o "$OUT/fuzz-elf-get-sections"
+	-o "$OUT/fuzz-libelf"
+
+$CC $CFLAGS \
+  -D_GNU_SOURCE -DHAVE_CONFIG_H \
+  -I. -I./lib -I./libelf -I./libebl -I./libdw -I./libdwelf -I./libdwfl -I./libasm \
+  -c "$SRC/fuzz-libdwfl.c" -o fuzz-libdwfl.o
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE fuzz-libdwfl.o \
+	./libasm/libasm.a ./libebl/libebl.a ./backends/libebl_backends.a ./libcpu/libcpu.a \
```

Comment:
I wonder why it's linked against these libraries? I opened https://github.com/google/oss-fuzz/pull/7412 where I dropped them but if it's intentional it would be great if there was a comment or something like that. Thanks!

Reply:
Got it. Thanks! I'll go ahead and close that PR then.

I think as long as issues like https://github.com/google/oss-fuzz/issues/7357 aren't triggered it should be possible to link all the fuzz targets with the common set of the libraries.

Reply:
Yep, sounds good and thanks for pointing https://github.com/google/oss-fuzz/issues/7357 out.